src/dev/build/args.ts

```diff
@@ -40,7 +40,9 @@ export function readCliArgs(argv: string[]) {
       'silent',
       'debug',
       'help',
-      'use-snapshot-epr',
+    ],
+    string: [
+      'epr-registry'
```

Comment:
Switched this to a string so you could specify either registry, regardless of what the default is.